packages/react/src/components/Tab/Tab.js

```diff
@@ -160,6 +160,7 @@ export default class Tab extends React.Component {
         {...other}
         tabIndex={-1}
         className={classes}
+        aria-disabled={disabled}
```

Comment:
How does this pair with `role="presentation"`?

Reply:
It does remove the DAP violation but as Vince said, since the error is coming from the anchor tag, the aria-disabled should live there so I'm gonna move it!